server/http.go

```diff
@@ -349,15 +334,14 @@ func (p *Plugin) handleStartMeeting(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	appErr = p.API.KVSet(fmt.Sprintf("%v%v", postMeetingKey, meetingID), []byte(createdPost.Id))
-	if appErr != nil {
+	if appErr = p.API.KVSet(fmt.Sprintf("%v%v", postMeetingKey, meetingID), []byte(createdPost.Id)); appErr != nil {
 		http.Error(w, appErr.Error(), appErr.StatusCode)
 		return
 	}
 
 	meetingURL := p.getMeetingURL(meetingID)
-	_, err = w.Write([]byte(fmt.Sprintf(`{"meeting_url": "%s"}`, meetingURL)))
-	if err != nil {
+
+	if _, err := w.Write([]byte(fmt.Sprintf(`{"meeting_url": "%s"}`, meetingURL))); err != nil {
```

Comment:
Non blocking: I'm wondering what the motivation behind these kind of changes is. Does it make the code more readable? Is there a idiomatic guide to follow here?